Ships as Raw.txt

```diff
@@ -0,0 +1,780 @@
+Anti Squadron|KK
+Upgrade|Title Officers TurboLaser Offensive Offensive  Weapon Team Ordance   
+FrontArcOffset|24.6
+//How far from the front corner does the hull zone line hit the cardboard edge//
+RearArcOffset|18.7
+//How far from the rear corner does the hull zone line hit the cardboard edge//
+FrontArcConjunction|52
+//How far from the front of the ship do the front hull zone lines meet//
+RearArcConjunction|41
+//How far from the rear of the ship do the rear hull zone lines meet//
+
+//This ship has a 9 mm section between the rear and front conjuctions.//
+51++
+Name|Victory I-class Star Destroyer
+Keyword|None
+Faction|Empire
+Size|Medium
+Points|73
+Hull|8
+Defense token| Rd Rd Br
+Command|3
+Squad|3
+Eng|4
+Nav chart 
+Speed 1|1
+Speed 2|0 1  
+Front Arc 3 |RRRKKK
+Side Arc 3 |RRK
+Rear Arc 1 |RR
+Anti Squadron|B
+Upgrade|Title Officers TurboLaser Offensive Weapon Team Ordance   
+FrontArcOffset|21
+//How far from the front corner does the hull zone line hit the cardboard edge//
+RearArcOffset|21
+//How far from the rear corner does the hull zone line hit the cardboard edge//
+FrontArcConjunction|47
+//How far from the front of the ship do the front hull zone lines meet//
+RearArcConjunction|54
+//How far from the rear of the ship do the rear hull zone lines meet//
+//This ship has a 1 mm section between the rear and front conjuctions.//
+
+Name|Victory II-class Star Destroyer
+Keyword|None
+Faction|Empire
+Size|Medium
+Points|85
+Hull|8
+Defense token| Rd Rd Br
+Command|3
+Squad|3
+Eng|4
+Nav chart 
+Speed 1|1
+Speed 2|0 1  
+Front Arc 3 |RRRBBB
+Side Arc 3 |RRB
+Rear Arc 1 |RR
+Anti Squadron|B
+Upgrade|Title Officers TurboLaser Offensive Weapon Team Ion cannon    
+FrontArcOffset|21
+//How far from the front corner does the hull zone line hit the cardboard edge//
+RearArcOffset|21
+//How far from the rear corner does the hull zone line hit the cardboard edge//
+FrontArcConjunction|47
+//How far from the front of the ship do the front hull zone lines meet//
+RearArcConjunction|54
+//How far from the rear of the ship do the rear hull zone lines meet//
+
+//This ship has a 1mm section between the rear and front conjuctions.//
+
+Name|Munificent-class Comms Frigate
+Keyword| Droid Bombard Comms
+Faction|SA
+Size|Medium
+Points|70
+Hull|6
+Defense token| Sal Rd Br
+Command|2
+Squad|3
+Eng|4
+Nav chart 
+Speed 1|1
+Speed 2|1 2
+Front Arc 4 |RRBB
+Side Arc 3 |RRR
+Rear Arc 2 |RR
+Anti Squadron|R
+Upgrade|Title Officers TurboLaser Offensive Support Team Ion Cannon  
+FrontArcOffset| 7.9
+//How far from the front corner does the hull zone line hit the cardboard edge//
+RearArcOffset|3.2
+//How far from the rear corner does the hull zone line hit the cardboard edge//
+FrontArcConjunction|53
+//How far from the front of the ship do the front hull zone lines meet//
+RearArcConjunction|49
+//How far from the rear of the ship do the rear hull zone lines meet//
+
+
+Name|Munificent-class Star Frigate
+Keyword| Droid Bombard Transport
+Faction|SA

```

Comment:
I added some factions to the squadron parses PR, if we can match those that would be awesome.

I can absolutely be talked into kicking this one down the road for now though.